src/java/org/apache/cassandra/service/StorageServiceMBean.java

```diff
@@ -308,7 +309,9 @@ public interface StorageServiceMBean extends NotificationEmitter
      * @param options map of options for cleanup operation, consult nodetool's ClearSnapshot
      * @param tag name of snapshot to clear, if null or empty string, all snapshots of given keyspace will be cleared
      * @param keyspaceNames name of keyspaces to clear snapshots for
+     * @deprecated See CASSANDRA-18111
      */
+    @Deprecated(since = "5.1")
```

Comment:
ah interesting, these move to its own MBean, something to keep in mind for Cassandra ANalytics